app/konnect/analytics/explorer.md

```diff
@@ -0,0 +1,116 @@
+
+All time interval presets are **relative**. 
+For custom reports, you can also choose a **custom** date range.
+
+* **Relative** time frames are dynamic and the report captures a snapshot of data
+relative to when a user views the report.
+* **Custom** time frames are static and the report captures a snapshot of data
+during the specified time frame. You can see the exact range below
+the time frame selector. For example:
+
+    ```
+    Jan 26, 2023 12:00 AM - Feb 01, 2023 12:00 AM (PST)
+    ```
+
+Interval | Description  
+---------|-------------
+Last 15 minutes | Data is aggregated in one minute increments.
+Last hour| Data is aggregated in one minute increments.
+Last six hours | Data is aggregated in one minute increments.
+Last 12 hours| Data is aggregated in one hour increments.
+Last 24 hours| Data is aggregated in one hour increments.
+Last seven days | Data is aggregated in one hour increments.
+Last 30 days | Data is aggregated in daily increments.
+Current week | Data is aggregated in one hour increments. Logs any traffic in the current calendar week. 
+Current month | Data is aggregated in one hour increments. Logs any traffic in the current calendar month. 
+Previous week | Data is aggregated in one hour increments. Logs any traffic in the previous calendar week.
+Previous month | Data is aggregated in daily increments. Logs any traffic in the previous calendar month. 
+
+{% endnavtab %}
+{% endnavtabs %}
+## Actions
+
+After customizing a view using Explorer's metrics and filters, you can perform several actions:
+
+* **Save as a Report**: This function creates a new custom report based on your current view, allowing you to revisit these specific insights at a later time.
+* **Export as CSV**: If you prefer to analyze your data using other tools, you can download the current view as a CSV file, making it portable and ready for further analysis elsewhere.
+
+## Diagnosing latency issues example
+
+Explorer in {{site.konnect_saas}} gives you the power to monitor your API data in detail and export that data to a CSV file. 
+Let's go through an example situation where you could leverage custom reports created in Explorer. With this type of report, you can start exploring which upstream service might cause the latency spike.
+
+One way you can build custom reports is by navigating to {% konnect_icon analytics %} **Analytics** in the {{site.konnect_short_name}} menu, then **Explorer**. This brings you to a page where you can control which analytics data you want to visualize. From here, configure the filter settings as follows:
+
+* **Name**: Production - Kong vs Upstream Latency (last hour)
+* **Show**: Line
+* **Date/Time**: Last One Hour
+* **With**: Kong Latency (avg), Upstream Latency (avg)
+* **Per**: Minute
+
+{:.note}
+> You can select more than one metric by clicking on **Select Multiple** next to the Metrics dropdown list.
+
+Then, they add a filter to filter by the control plane
+
+* **Filter By**: Control Plane
```

Comment:
Ah. I forgot to add the filter into my image :D I'll send you an updated version.